ui/src/actionBar.tsx

```diff
@@ -66,3 +69,24 @@ export default function ActionBar(props: ActionBarProps) {
     </Box>
   );
 }
+
+let setBackURL: ((url?: string) => void) | undefined;
```

Comment:
i know you have a comment, but this should just be something the hook gets from the context.  any reason to not just do this?

Reply:
That will touch a bunch of other paces in the code, including refactoring the multiple action bar instances into a single location and also providing a way to set the title in each location.